### PR TITLE
Clean up displacy port-related error messages, docs

### PR DESCRIPTION
**Pull request by polm**
## Description

There were some issues in the error messages and docs in #11948.

1. the error messages didn't specify the port argument to displacy.serve correctly
2. the docs didn't mark the auto select argument as new

This addresses those issues.

### Types of change

docs fix

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

**Comment by polm** [comment]
Python 3.8 Mac tests are failing with a strange error that looks unrelated to these changes:

```
Assertion failed: (timeout != -1), function uv__io_poll, file ../deps/uv/src/unix/kqueue.c, line 256.
tests/lang/lt/test_text.py ...............
##[error]Exit code 134 returned from process: file name '/Users/runner/runners/2.213.2/externals/node16/bin/node', arguments '"/Users/runner/work/_tasks/CmdLine_d9bafed4-0b18-4f58-968d-86655b4d2ce9/2.212.0/cmdline.js"'.
```

I'll try re-running it.

**Comment by polm** [comment]
Tests are fine after re-running, so this should be good to go.

File: website/docs/api/top-level.md [hunk 1]
```diff
@@ -237,17 +237,17 @@ browser. Will run a simple web server.
...
-| `auto_select_port` | If `True`, automatically switch to a different port if the specified port is already in use. Defaults to `False`. ~~bool~~                                          |
+| Name                                            | Description                                                                                                                                                       |
+| ----------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `docs`                                          | Document(s) or span(s) to visualize. ~~Union[Iterable[Union[Doc, Span]], Doc, Span]~~                                                                             |
+| `style`                                         | Visualization style, `"dep"`, `"ent"` or `"span"` <Tag variant="new">3.3</Tag>. Defaults to `"dep"`. ~~str~~                                                      |
```
Comment (rmitsch) [inline website/docs/api/top-level.md:243]: Somewhat unrelated, but `<Tag variant="new">3.3</Tag>` shouldn't be in `Description`, but in `Name`, right?
```suggestion
| `style` <Tag variant="new">3.3</Tag>| Visualization style, `"dep"`, `"ent"` or `"span"`. Defaults to `"dep"`. ~~str~~                                                      |
```

**Review by rmitsch — COMMENTED** [review]
LGTM. Only remark is an unrelated issue with docs in the same file that we could fix (if it's an issue indeed) while we're at it.

File: website/docs/api/top-level.md (hunk 1 above)
Comment (polm) [reply website/docs/api/top-level.md:243]: Yes, good catch.

File: website/docs/api/top-level.md [hunk 2]
```diff
@@ -237,17 +237,17 @@ browser. Will run a simple web server.
...
+| `options`                                       | [Visualizer-specific options](#displacy_options), e.g. colors. ~~Dict[str, Any]~~                                                                                 |
+| `manual`                                        | Don't parse `Doc` and instead expect a dict or list of dicts. [See here](/usage/visualizers#manual-usage) for formats and examples. Defaults to `False`. ~~bool~~ |
+| `port`                                          | Port to serve visualization. Defaults to `5000`. ~~int~~                                                                                                          |
+| `host`                                          | Host to serve visualization. Defaults to `"0.0.0.0"`. ~~str~~                                                                                                     |
+| `auto_select_port` <Tag variant="new">3.5</Tag> | If `True`, automatically switch to a different port if the specified port is already in use. Defaults to `False`. ~~bool~~                                        |
```
Comment (svlandeg) [inline website/docs/api/top-level.md:250]: Looks like this needs one more round of `prettier`

**Review by svlandeg — APPROVED** [review]
Should be good to merge after formatting!

**Comment by polm** [comment]
This showed a conflict in the Github UI, but a local merge finished without issue. I'll let the tests run just in case.

**Pull request merged**